### PR TITLE
IASI test YAMLs and minor changes to scripts

### PR DESCRIPTION
**Pull request by ADCollard**
Create test YAMLs for IASI and make some small changes to plotting and run scripts.

Testing of the IASI YAML is documented in [JEDI-T2O Issue #46](https://github.com/NOAA-EMC/JEDI-T2O/issues/46).

The following files need to be added to the obs/geovals directories as a full cycle of observations cannot be run on one processor for IASI and CrIS:

```
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/geovals/cris-fsr_n20_geoval_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/geovals/cris-fsr_npp_geoval_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/geovals/iasi_metop-a_geoval_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/geovals/iasi_metop-b_geoval_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/obs/cris-fsr_n20_obs_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/obs/cris-fsr_npp_obs_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/obs/iasi_metop-a_obs_2021080100_subset.nc4
/scratch1/NCEPDEV/da/Andrew.Collard/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20230217/obs/iasi_metop-b_obs_2021080100_subset.nc4
```

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
@emilyhcliu is the host of the files going forward, right? I will wait to test/approve until she can copy those files to the correct locations.

**Comment by ADCollard** [comment]
> @emilyhcliu is the host of the files going forward, right? I will wait to test/approve until she can copy those files to the correct locations.

@CoryMartin-NOAA Correct

**Comment by ADCollard** [comment]
Pushed CrIS_NPP and CrIS_N20 yamls.   This references [JEDI-T2O Issue 43](https://github.com/NOAA-EMC/JEDI-T2O/issues/43).

**Comment by ADCollard** [comment]
Updated `run_ufo_hofx_test.sh` to point at Emily's latest test data.

I'm ready for final reviews @CoryMartin-NOAA @emilyhcliu

File: ush/ufoeval/run_ufo_hofx_test.sh [hunk 1]
```diff
@@ -70,13 +74,18 @@ else
    exit 1
 fi
 
+if [ keep_output = YES ]; then
```
Comment (CoryMartin-NOAA) [inline ush/ufoeval/run_ufo_hofx_test.sh]: `$keep_output`?

Comment (ADCollard) [reply ush/ufoeval/run_ufo_hofx_test.sh]: @CoryMartin-NOAA Should be fixed now

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**